storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -199,6 +199,12 @@
     public static final Object TOPOLOGY_TUPLE_SERIALIZER_SCHEMA = String.class;
 
     /**
+     * Disable load aware grouping support.
+     */
+    public static final String TOPOLOGY_DISABLE_LOADAWARE = "topology.disable.loadaware";
```

Comment:
Could we call this something like `TOPOLOGY_DISABLE_LOADAWARE_MESSAGING` so it is clear what ceases to be aware of load?